vscode/src/chat/initialContext.ts

```diff
@@ -161,7 +161,7 @@ function getCurrentFileOrSelection({
                     if (range) {
                         items.push({
                             ...contextFile,
-                            type: 'current-selection',
+                            type: 'file',
```

Comment:
```suggestion
                            type: 'current-selection',
```

If there is range shouldn't this be the Context Item with current selection type? The current-selection type is important because that's when we rank the priority of each context item when we construct the final prompt message

Reply:
If we do that than the selection appears as @current-selection instead of showing the file:range even when we dynamically modify the initial selection and when we select (current selection) from the mention dropdown